### PR TITLE
Containers concept intro

### PR DESCRIPTION
**Pull request by crixo**
translation for /docs/concepts/containers overview

related to the issue #20958

### CODE REVIEW

File: content/it/docs/concepts/containers/_index.md [hunk 1]
```diff
@@ -1,4 +1,30 @@
...
+<!-- overview -->
+
+Ogni _container_ che viene eseguito è riproducibile; la pratica di includere le dipendenze all'interno di ciascuno _container_ permette di ottenere sempre lo stesso risultato ad ogni esecuzione del medesimo _container_. 
+
+I _Container_ permettono di disaccoppiare le applicazioni dall'infrastruttura del host su cui vengono eseguite. Questo approccio rende più facile il _deployment_ su cloud o sitemi operativi differenti tra loro.
+
+<!-- body -->
+
+## Immagine di container
+L'[immagine di un container](/docs/concepts/containers/images/) e' un pacchetto software pronto per essere eseguito che contiene tutto ciò che serve per eseguire un'applicazione: il codice sorgente e ciascun _runtime_ necessario, librerie applicative e di sistema, e le impostazioni predefinite per ogni configurazione necessaria.
```
Comment (fabriziopandini) [inline content/it/docs/concepts/containers/_index.md:18]: nit for making the sentence more readable
```suggestion
L'[immagine di un container](/docs/concepts/containers/images/) e' un pacchetto software che contiene tutto ciò che serve per eseguire un'applicazione: il codice sorgente e ciascun _runtime_ necessario, librerie applicative e di sistema, e le impostazioni predefinite per ogni configurazione necessaria.
```

Comment (crixo) [reply content/it/docs/concepts/containers/_index.md]: done. Waiting for final ok to squash

**Comment by fabriziopandini** [comment]
Looks great! Please squash commits

**Comment by crixo** [comment]
> Looks great! Please squash commits

squashed

**Pull request merged**